modules/proxistoreBidAdapter.js

```diff
@@ -1,11 +1,24 @@
-const { registerBidder } = require('../src/adapters/bidderFactory.js');
+
```

Comment:
Can you remove this empty line? It is not common practice to start .js file with empty line.